utils/ponoresult.h

```diff
@@ -0,0 +1,33 @@
+/*********************                                                        */
+/*! \file
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Ahmed Irfan, Makai Mann, Florian Lonsing
+ ** This file is part of the pono project.
+ ** Copyright (c) 2019, 2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file LICENSE in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief
+ **
+ **
+ **/
+
+#pragma once
+
+#include <string>
+
+namespace pono {
+
+typedef enum
+{
+  PROPERTY_TRUE = 0,
```

Comment:
It definitely makes sense to avoid magic constants, but I'm wondering if we should just unify this and ProverResult instead of having two separate classes: https://github.com/upscale-project/pono/blob/e25fb8fa112eeb2b85427fda9475d9b247994654/core/proverresult.h#L25